.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI/CD Workflow
+
+on:
+  push:
+    branches:
+  pull_request:
+    branches:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run lint
```

Comment:
Missing script command --> check fails 
<img width="1227" alt="Screenshot 2024-10-31 at 16 25 51" src="https://github.com/user-attachments/assets/b8ce02a6-dc9e-4328-87e3-667c185d2676">

Reply:
Add a lint command in package.json

Reply:
Done.

Reply:
Still seeing errors 
<img width="856" alt="Screenshot 2024-10-31 at 16 44 17" src="https://github.com/user-attachments/assets/fc6351b9-0bd1-4270-adac-0613ef645bb7">
<img width="1273" alt="Screenshot 2024-10-31 at 16 43 33" src="https://github.com/user-attachments/assets/836de0c3-0206-4307-818b-a1830c32a85c">